### ISSUE TITLE
Building ngeo on windows

### ISSUE THREAD

**Issue by monodo** [camptocamp/ngeo#2482]
Building ngeo on windows fails with following error:
@fredj : seems that you talk about this issue during workshop but can't remember the solution
```node buildtools/build.js .build/ngeo.json dist/ngeo.js
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: spawn c:\Projets\ngeo\node_modules\.bin\jsdoc ENOENT
    at exports._errnoException (util.js:1034:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
Makefile_windows.mak:288: recipe for target 'dist/ngeo.js' failed
make: *** [dist/ngeo.js] Error 1```

**Comment by fredj** [comment]
During the workshop, it was an error with java, this time the error seems to come from jsdoc

**Comment by kalbermattenm** [comment]
Actually, the error comes from the following command:
``` 
node buildtools/build.js .build/ngeo.json dist/ngeo.js
```
If I did get it right, the weird thing is that this error (spawn) usually means "I cannot find this command", but as far as I can see, "c:\Projets\ngeo\node_modules\.bin\jsdoc" exists and we do not really get where to start the debug process...

**Comment by monodo** [comment]
Actually, I'm not sure the issue comes from windows & jsdoc: I just built ol3 and it run nice and smooth...

**Comment by kalbermattenm** [comment]
The issue is not in ol3... It has to do with something in ngeo... but I do not know how to find from where it comes...

**Comment by monodo** [comment]
That's the point

**Comment by kalbermattenm** [comment]
OK... I made a step forward...

By Modifying, <a href=https://github.com/camptocamp/ngeo/blob/master/buildtools/generate-info.js#L158 target=_blank>this</a>:
```javascript
let child = spawn(jsdoc, ['-c', jsdocConfig].concat(newerSources));
```
Into this:
```javascript
let child = spawn(jsdoc, ['-c', jsdocConfig].concat(newerSources), {shell: true});
```
Reference:
https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows

It does not throw the error (`Error: spawn c:\Projets\ngeo\node_modules\.bin\jsdoc ENOENT`) anymore, but then I get another one...:
```
ERR! Trouble generating info: La ligne de commande est trop longue.
```
Effectively, `newerSources` is a huge JS array...

**Comment by fredj** [comment]
fixed with #2536
